Trappist/src/Promact.Trappist.Repository/Questions/QuestionRepository.cs

```diff
@@ -157,21 +153,69 @@ public QuestionRepository(TrappistDbContext dbContext)
 
         public async Task UpdateSingleMultipleAnswerQuestionAsync(int questionId, QuestionAC questionAC, string userId)
         {
-            var updatedQuestion = Mapper.Map<QuestionDetailAC, Question>(questionAC.Question);
-            var options = questionAC.SingleMultipleAnswerQuestion.SingleMultipleAnswerQuestionOption;
+            var updatedQuestion = await _dbContext.Question.FindAsync(questionId);
+            await _dbContext.Entry(updatedQuestion).Reference(x => x.SingleMultipleAnswerQuestion).LoadAsync();
+            await _dbContext.Entry(updatedQuestion.SingleMultipleAnswerQuestion).Collection(x => x.SingleMultipleAnswerQuestionOption).LoadAsync();
+            var option = updatedQuestion.SingleMultipleAnswerQuestion.SingleMultipleAnswerQuestionOption;
```

Comment:
please make this variable name descriptive. you can use singleMultipleQuestionAnswerOption or questionAnswerOption